src/libs/actions/Task.ts

```diff
@@ -587,6 +587,9 @@ function editTaskAssignee(report: OnyxTypes.Report, ownerAccountID: number, assi
     };
     const successReport: NullishDeep<OnyxTypes.Report> = {pendingFields: {...(assigneeAccountID && {managerID: null})}};
 
+    const additionalReport: NullishDeep<OnyxTypes.Report> = {}
+    report.reportID === currentReportID && (additionalReport.lastReadTime = DateUtils.getDBTimeWithSkew());
```

Comment:
We have lint error here.

![Screenshot 2024-07-29 at 18 46 50](https://github.com/user-attachments/assets/85655f3c-8d66-4ce1-8ff4-bf2c744dd9e6)

Maybe we can change it to 
```js
const additionalReport: NullishDeep<OnyxTypes.Report> = report.reportID === currentReportID ? {lastReadTime: DateUtils.getDBTimeWithSkew()} : {}
```

Reply:
What about this version?

```ts
const additionalReport: NullishDeep<OnyxTypes.Report> = {};
    if (report.reportID === currentReportID) {
        additionalReport.lastReadTime = DateUtils.getDBTimeWithSkew();
    }
```

Reply:
That works too!

Reply:
fixed